script.domoticz.scenes/resources/lib/script.py

```diff
@@ -0,0 +1,307 @@
+import xbmcplugin
+import xbmcgui
+import xbmc
+import xbmcaddon
+import os
+import sys
+import requests
+import json
+
+addon_handle = int(sys.argv[1])
+xbmcplugin.setContent(addon_handle, 'videos')
+addonID = 'script.domoticz.scenes'
+addonVersion = '0.0.8'
+addonDate = "24 Maart 2021"
+
+__addon__ = xbmcaddon.Addon()
+__addonname__ = __addon__.getAddonInfo('name')
+__icon__ = __addon__.getAddonInfo('icon')
+
+LIB_DIR = xbmc.translatePath(os.path.join(xbmcaddon.Addon(id=addonID).getAddonInfo('path'), 'resources', 'lib'))
+sys.path.append(LIB_DIR)
```

Comment:
Sys.path.append is a bad practice in Kodi add-ons, please use relative module imports

Reply:
removed